charts/kubefed/values.yaml

```diff
@@ -62,6 +62,8 @@ controllermanager:
 
   certManager:
     enabled: false
+    apiGroup: certmanager.k8s.io
+    apiVersion: v1alpha1
```

Comment:
@jimmidyson I agree we could bump the default values, however I'd say the customization of `apiGroup` or `apiVersion` is exposed in multiple helm charts nowadays, e.g prometheus-operator. So I don't find it to be a negative change to be honest.

Reply:
Could you link to the `prometheus-operator` allows customization of `apiGroup` and/or `apiVersion`? i can't seem to find it - but i'm never very good at looking for this kind of thing :)

Reply:
Generally, allowing `apiVersion` and `apiGroup` configurable could introduce problems with the resource definitions not being applicable (API version changes generally imply API changes). It might be safe in this case, but I'd prefer to err on the side of caution and not allow it to be configurable in case of introducing issues in future.

Reply:
I could cite `drone, etcd-operator, externa-dns, couchbase-operator, concourse, g2, jenkins, clamav,` as other stable charts which expose the apiVersion to be configurable for any of their resources.

Reply:
Would you be able to link to one of those for reference please?

Reply:
Considering this again, it would make more sense to use helm's `Capabilities.APIVersions.Has` to set the correct API group/version in the Certificate` resource. I'll add that suggestion to the relevant place in this PR, but now I'm strongly advocating for removing this manual values setting here.

```suggestion
```

Reply:
See https://github.com/kubernetes-sigs/kubefed/pull/1288#pullrequestreview-500252794.